modules/administration/pages/ssl-certs-selfsigned.adoc

```diff
@@ -1,9 +1,16 @@
-[[custom-ssl]]
-= Custom SSL Certificates
+[[ssl-certs-selfsigned]]
+= Self-Signed SSL Certificates
 
-You can use custom certificates with {productname} and {productname} Proxy.
+By default, {productname} uses a self-signed certificate.
+In this case, the certificate is created and signed by {productname}.
+This method does not use a third party certificate authority to guarantee that the details of the certificate are correct.
```

Comment:
This is not wong, but I would like to add that the and independent party check the details.

```suggestion
This method does not use an independent third party certificate authority to guarantee that the details of the certificate are correct.
```

The background: If you use it internally only, why do you need an "independent" authority to check the details?
I would say nobody is so trustworthy than yourself, isn't it?

Reply:
This is mentioned in the intro, but I'll copy it across.